BUILDING.md

```diff
@@ -160,7 +158,7 @@ The forthcoming OpenSSL-1.1.1 will have different requirements. Please refer to
 * Python 2.6 or 2.7
 * GNU Make 3.81 or newer
 
-On macOS, you will need to install the `Xcode Command Line Tools` by running
+On macOS, install the `Xcode Command Line Tools` by running
```

Comment:
~the~?

Reply:
I think this was resolved by a fix-up commit? If not, unresolve or comment again or something...

Reply:
```suggestion
On macOS, install `Xcode Command Line Tools` by running
```

Reply:
Sure, done.